src/utils/azure.ts

```diff
@@ -105,29 +106,28 @@ export async function getEventHubsConnectionString(context: IEventHubsConnection
     };
 }
 
-// Todo: Uncomment out in future PR
-// export async function getSqlDatabaseConnectionString(context: ISqlDatabaseConnectionWizardContext): Promise<IResourceResult> {
-//     const serverName: string = nonNullValue(context.sqlServer?.name);
-//     const dbName: string = nonNullValue(context.sqlDatabase?.name);
-//     const username: string | undefined = context.sqlServer?.administratorLogin;
-
-//     if (!username) {
-//         throw new Error(localize('unableToDetermineSqlConnection', 'We were unable to locate the admin user for this SQL server, please add these credentials to your resource to proceed.'));
-//     }
-
-//     let password: string | undefined = context.newSqlAdminPassword;  // password is never returned back to us on the sqlServer object
-//     if (!password) {
-//         password = (await context.ui.showInputBox({
-//             prompt: localize('sqlPasswordPrompt', 'Please enter your SQL server\'s admin password.'),
-//             password: true
-//         })).trim() ?? 'null';
-//     }
-
-//     return {
-//         name: dbName,
-//         connectionString: `Server=${serverName}.database.windows.net,1433;Database=${dbName};User=${username};Password=${password}`
-//     };
-// }
+export async function getSqlDatabaseConnectionString(context: ISqlDatabaseConnectionWizardContext): Promise<IResourceResult> {
+    const serverName: string = nonNullValue(context.sqlServer?.name);
+    const dbName: string = nonNullValue(context.sqlDatabase?.name);
+    const username: string | undefined = context.sqlServer?.administratorLogin;
+
+    if (!username) {
+        throw new Error(localize('unableToDetermineSqlConnection', 'We were unable to locate the admin user for this SQL server, please add these credentials to your resource to proceed.'));
+    }
+
+    let password: string | undefined = context.newSqlAdminPassword;  // password is never returned back to us on the sqlServer object
+    if (!password) {
```

Comment:
I don't really know of any way to do a call to check the password like that.  I think if the user's connection string is basically right except for the password, it would be fairly easy to just go into the `local.settings.json` and correct the password manually?

Also this scenario we're worried about is pretty rare.  You won't ever run into this issue on initial create because we already have the password stored in-memory and can just immediately populate if necessary.  This only really is an issue if you deleted the string and then hit F5 and then proceeded to mis-enter your password.

Reply:
I feel like connection strings are almost always copy/pasted, not typed. So the likelihood of a mistyped password is pretty low.

Reply:
I think that this is prompt is for the SQL admin password which is used to generate the connection string though.

But as Matt said, as long as they created the project through the extension, they shouldn't have to retype it. Though that does bring a mild concern-- where is the password being stored in-memory?

Reply:
Just on the context when it's initially created as `context.newSqlAdminPassword`.  I started discussing this briefly with you, but maybe we need to add this to `context.valuesToMask` as well as potentially run with `callWithMaskHandling`?